### PR TITLE
Development

### PR DESCRIPTION
**Pull request by btuerker**
Project is ended.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,373 @@
...
+                    </li>
+                    <li>
+                        <img src="./assets/user-icon.png" alt="">
+                        <span style="padding-top:10px;">
+                            <span style="font-weight:bold;">UserUnknown</span> &nbsp;&nbsp;&nbsp;
+                            <span style="color:gray;">10 months ago </span>
+                            <br />
+                            Some comment text we have here
+                            <br />
+                            <div class="like-buttons">
```
Comment (heyalvaro) [inline index.html]: Element div not allowed as child of element span in this context

File: index.html [hunk 2]
```diff
@@ -0,0 +1,373 @@
...
+                <div class="playlist-title">
+                    Mix - Lake of Tears - To Blossom Blue
+                </div>
+                <span class="playlist-youtube-badge">YouTube</span>
+                <span class="playlist-save-icon">
+                    <img src="./assets/playlist-save-icon.png" alt="">
+                </span>
+                <ul class="playlist-panel">
+                    <li class="playlist-panel-content">
+                        <img src="https://i.ytimg.com/vi/wEcEjNsn1Io/hqdefault.jpg?sqp=-oaymwEYCKgBEF5IVfKriqkDCwgBFQAAiEIYAXAB&rs=AOn4CLBG0U1Tgk6jcEGey9XAyf9IuFkbYw"
```
Comment (heyalvaro) [inline index.html]: Images are not loading. Is the thing that comes after the `.jpg?` necessary?

File: css/styles.css [hunk 3]
```diff
@@ -0,0 +1,263 @@
...
+            height:15px;
+            padding-top:5px;
+        }
+        .recomendations {
+            grid-column: 2;
+            grid-row: 2;
+        }
+        .recomendations ul {
+            list-style-type: none;
+            padding-left: 0px;
```
[the remark this answers is not in the document]
Comment (btuerker) [reply css/styles.css]: we used it because we added text-decoration:none to unordered list

File: css/styles.css [hunk 4]
```diff
@@ -1,9 +1,4 @@
-body {
-    background-color: white;
-}
-
 .navbar {
-    background-color: white;
     width: 100%;
     height: 56px;
     padding: 0px 16px 0px 16px;
```
Comment (heyalvaro) [inline css/styles.css]: You can omit the `px` when the value is 0

**Pull request merged**